tests/roles/glance_adoption/files/glance_swift.yaml

```diff
@@ -0,0 +1,37 @@
+spec:
+  glance:
+    enabled: true
+    apiOverrides:
+      default:
+        route: {}
+    template:
+      databaseInstance: openstack
+      storageClass: "local-storage"
+      storageRequest: 10G
+      customServiceConfig: |
+        [DEFAULT]
+        enabled_backends = default_backend:swift
+        [glance_store]
+        default_backend = default_backend
+        [default_backend]
+        swift_store_create_container_on_put = True
+        swift_store_auth_version = 3
+        swift_store_auth_address = {{ .KeystoneInternalURL }}
```

Comment:
AFAICT KeystoneInternalURL and ServicePassword are not defined during the playbook run. ServicePassword might be replaced with glance_password though